app/views/posts/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Post Details</h1>
+<p>Title: </p>
+<p>Content: </p>
+<p>Author: </p>
```

Comment:
- [ ] Similarly, let's add a corresponding _view_ (`# app/views/posts/index.html.erb`) to match the `index` actions in the _posts_ controller.